### ISSUE TITLE
cmdLineTester_SCURLClassLoaderTests fails on Windows

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#2351]
Seems to have started on July 6.

https://ci.eclipse.org/openj9/job/Test-Sanity-JDK8-win_x86-64_cmprssptrs/140/
https://ci.eclipse.org/openj9/job/Test-Sanity-JDK8-win_x86/20/
https://ci.eclipse.org/openj9/job/Test-Sanity-JDK10-win_x86-64_cmprssptrs/148/

```
cmdLineTester_SCURLClassLoaderTests_1
===============================================
Running test cmdLineTester_SCURLClassLoaderTests_1 ...
===============================================
cmdLineTester_SCURLClassLoaderTests_1 Start Time: Fri Jul  6 17:40:30 2018 Epoch Time (ms): 1530916830832
test with Mode610
Parsing exclude list...
  add excludes for [all]
  add excludes for [win_x86-64_cr]
  add excludes for [latest]
  add excludes for [SE100]
DONE in 0 ms.
*** Starting test suite: Shared Classes URLClassLoaderTests Suite - Windows Configuration ***
 
#######################################################
Running tests in mode 210 with command line options:  
#######################################################
 
Executing command: "C:/Users/jenkins/workspace/Test-Sanity-JDK10-win_x86-64_cmprssptrs/jdk/bin/java"  -Xcompressedrefs -Xcompressedrefs -Xjit -Xgcpolicy:gencon  -Xshareclasses:destroyAll

Executing command: "C:/Users/jenkins/workspace/Test-Sanity-JDK10-win_x86-64_cmprssptrs/jdk/bin/java"  -Xcompressedrefs -Xcompressedrefs -Xjit -Xgcpolicy:gencon  -Xshareclasses:persistent,controlDir=.,name=invalidCacheName,destroyAll

Testing: Sanity.BuildInitial
Test start time: 2018/07/06 17:40:31 Central Standard Time
Running command: cmd /c ./Sanity/buildInitial.bat C:/Users/jenkins/workspace/Test-Sanity-JDK10-win_x86-64_cmprssptrs/jdk/bin/../bin
Time spent starting: 4 milliseconds
Time spent executing: 16 milliseconds
Test result: FAILED
 [ERR] '.' is not recognized as an internal or external command,
 [ERR] operable program or batch file.
>> Success condition was not found: [Output match: A.jar]
>> Failure condition was not found: [Output match: Usage:]
>> Failure condition was not found: [Output match: no such]
>> Failure condition was not found: [Output match: Exception:]
>> Failure condition was not found: [Output match: Error:]```

@hangshao0 @llxia fyi

blocking omr acceptance from promoting

**Comment by llxia** [comment]
I think it is related to PR #1957. We cannot change D globally.

https://github.com/eclipse/openj9/blob/bea496f20be72b37b4c4f26f08c54f803a2207d3/test/TestConfig/settings.mk#L103-L107

fyi @Ailloviee

**Comment by pshipton** [comment]
@llxia there is a Revert button you can use in #1957.

**Comment by pshipton** [comment]
Assuming this is fixed by #2354
